runtime/doc/vietnamese.txt

```diff
@@ -0,0 +1,87 @@
+*vietnamese.txt*   Vim
+
+
+		  VIM REFERENCE MANUAL    by Phạm Bình An
+
+
+Vietnamese language localization and support in Vim	  *vietnamese* *Vietnamese*
+
+                                      Type |gO| to see the table of contents.
```

Comment:
that is not in Vim.

Reply:
Can you please suggest how I should change that line?

Reply:
I am talking about `gO`

Reply:
OK, I have manually added table of contents to vietnamese.txt